sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -222,6 +225,61 @@ public async Task StartRecognizeReceiptsPopulatesExtractedReceipt(bool useStream
             Assert.That((float?)receipt.Total, Is.EqualTo(1203.39).Within(0.0001));
         }
 
+        /// <summary>
+        /// Verifies that the <see cref="FormRecognizerClient" /> is able to connect to the Form
+        /// Recognizer cognitive service and perform analysis based on a custom labeled model.
+        /// </summary>
+        [Test]
+        [TestCase(true)]
+        [TestCase(false, Ignore = "The form has not been uploaded to GitHub yet.")]
+        public async Task StartRecognizeCustomFormsLabeled(bool useStream)
+        {
+            var client = CreateInstrumentedClient();
+
+            FormTrainingClient trainingClient = client.GetFormTrainingClient();
```

Comment:
Let's creating an instance of the TrainingClient directly here, as that's the preferred approach.

Reply:
Would it also be possible to either move the training code to a separate method, an initialization function, or add a comment that the training section is a prerequisite to the model, but not the functionality being tested here?  Just in case customers use this as sample code, I'd like it to be clear that training doesn't have to happen in our SDK, it could (and most likely will) happen in a UI tool like the [Form Recognizer's Labeling Tool](https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/quickstarts/label-tool).

Reply:
will do

Reply:
Looks like SetUp applies to all methods in the test class, which we don't need. So I created an initialization function instead. I also added comment referring to the labeling tool.

Reply:
Did you see [this PR](https://github.com/Azure/azure-sdk-for-net/pull/11410)?  IIRC, there are some examples of initialization prior to tests in there.

Reply:
Merging for now. If we need to change it, I will do that later